frontend/src/layout/TopContent/CommandPaletteButton.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { commandPaletteLogic } from 'lib/components/CommandPalette/commandPaletteLogic'
+import { SearchOutlined } from '@ant-design/icons'
+import { platformCommandControlKey } from 'lib/utils'
+
+export function CommandPaletteButton(): JSX.Element {
+    const { isPaletteShown } = useValues(commandPaletteLogic)
+    const { showPalette } = useActions(commandPaletteLogic)
+
+    return (
+        <span
+            data-attr="command-palette-toggle"
+            className="btn btn-sm btn-light hide-when-small"
+            onClick={showPalette}
+            title={isPaletteShown ? 'Hide Command Palette' : 'Show Command Palette'}
+        >
+            <SearchOutlined size={1} style={{ marginRight: '0.5rem' }} />
+            {platformCommandControlKey()} + K
```

Comment:
This is quite a nit, but this looks better imo:
![image](https://user-images.githubusercontent.com/53387/95861891-622cee80-0d62-11eb-8a1a-423b92fb4d22.png)